src/styles/blocks/container.scss

```diff
@@ -0,0 +1,9 @@
+.container {
```

Comment:
you can use `container` once and don't add it all over the page in separate elements.